src/main/scala/ch/epfl/bluebrain/nexus/kg/async/RevisionedId.scala

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.bluebrain.nexus.kg.async
+
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+
+/**
+  * A type class of ''A'' which has the methods ''id'' and ''rev''
```

Comment:
_A type class that provides a specific revision identifier (''id: AbsoluteIri'' and ''rev: Long'') of a resource of type ''A''._